data/siteMetadata.js

```diff
@@ -22,6 +22,15 @@ const siteMetadata = {
     simpleAnalytics: false, // true or false
     googleAnalyticsId: '', // e.g. UA-000000-2 or G-XXXXXXX
   },
+  newsletter: {
+    provider: 'buttondown', // supported providers: mailchimp
+    mailChimpConfig: {
+      apiKey: process.env.NEXT_PUBLIC_MAILCHIMP_API_KEY,
+      apiServer: process.env.NEXT_PUBLIC_MAILCHIMP_API_SERVER,
+      audienceId: process.env.NEXT_PUBLIC_MAILCHIMP_AUDIENCE_ID,
+    },
+    buttonDown: process.env.NEXT_PUBLIC_BUTTONDOWN,
```

Comment:
How about `buttondownConfig` for consistency? And we can change `disqus` below to `disqusConfig` as well